pkg/util/exec/exec.go

```diff
@@ -170,7 +173,7 @@ func (*CommandExecutor) ExecuteCommandWithOutputFileTimeout(timeout time.Duratio
 	}
 
 	if err != nil {
-		return string(cmdOut), err
+		return string(cmdOut), errors.Errorf("%s. %s", err.Error(), string(cmdOut))
```

Comment:
We have to be careful wrapping the error at this layer. Sometimes the caller need to look at the error type, so this would mask the inner error.  For example, anywhere `ExitStatus(err)` is called. The only solution I can think of is to have the caller append the `cmdOut` to the error when it returns the error, but that involves a lot more places to touch.

Reply:
That's what I like with this option, we don't need to do anything on the above layers. Let me look into at using a proper error type then.